docs/api-reference/next/image.md

```diff
@@ -202,41 +252,60 @@ Try it out:
 - [Demo the shimmer effect with `blurDataURL` prop](https://image-component.nextjs.gallery/shimmer)
 - [Demo the color effect with `blurDataURL` prop](https://image-component.nextjs.gallery/color)
 
-## Advanced Props
+#### `style`
 
-In some cases, you may need more advanced usage. The `<Image />` component optionally accepts the following advanced properties.
+Allows [passing CSS styles](https://reactjs.org/docs/dom-elements.html#style) to the underlying image element.
 
-### style
+```jsx:components/ProfileImage.js
+const imageStyle = {
+  borderRadius: '50%',
+  border: '1px solid #666',
```

Comment:
Might want to use something other than 666 to make this palatable to a wider audience